### PR TITLE
Make it easier to generate a GAIA report without a testsuite run

### PR DESCRIPTION
**Pull request by pmderodat**

### CODE REVIEW

File: tests/tests/test_report_gaia.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    index_dir = tmp_path / "index"
+    report_dir = tmp_path / "report"
+    index_dir.mkdir()
+    report_dir.mkdir()
+
+    # Write test entries on disk
+    index = create_report(results, str(index_dir))
+
+    # Generate the GAIA report from them
+    dump_gaia_report(index, str(report_dir))
```
Comment (hainque) [inline tests/tests/test_report_gaia.py:23]: Very minor: I was first surprised to see a report_dir variable
and a call to "create_report" passing another dir as an argument.

Maybe create_report_index?

**Review by hainque — DISMISSED** [review]
Only a very minor comment on the name of a "utils" function, certainly not blocking.

File: tests/tests/test_report_gaia.py (hunk 1 above)
Comment (pmderodat) [reply tests/tests/test_report_gaia.py:23]: I’ll do some rewordings, but not the ones suggested ;-)

`create_report` does create a report (the YAML one, i.e. the standard one for e3-testsuite), including its index. The other report that is generated is the GAIA-compatible one. So I think renaming `report_dir` to `gaia_report_dir` is the best way to resolve the ambiguity.

**Comment by hainque** [comment]
Ok, works for me now. Thanks!

**Comment by pmderodat** [comment]
Ok, thank you for the review! I’ll will wait for the end of the stable bump before merging…

**Review by hainque — APPROVED** [review]
Looks good now, and I was able to use it easily from a non e3 testsuite context indeed.
Thanks!

**Comment by pmderodat** [comment]
Thanks, I’m glad it was useful!

**Pull request merged**